Src/BootCamp.Chapter1/ArrayOperations.cs

```diff
@@ -88,8 +195,33 @@ public static int[] InsertLast(int[] array, int number)
         /// <returns>A new array with element inserted at a given index. If an array is empty or null, returns new array with number in it.</returns>
         public static int[] InsertAt(int[] array, int number, int index)
         {
-            // ToDo: implement.
-            return array;
+            if (index < 0 || index > array.Length)
+                return array;
+            if (array == null || array.Length == 0)
+                return new int[] { number };
+
+            int[] newArray = new int[array.Length + 1];
+
+            newArray[index] = number;
+
+            bool isAfterIndex = false;
+
+            for (int i = 0; i < array.Length ; i++)
+            {
+                if (i == index)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Add curly braces around the nested statement(s) in this 'if' block.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=4886613)